### PR TITLE
Implements coala-ci

### PR DESCRIPTION
**Pull request by Arsfiqball**
This Pull Request is being done as part of the Google Code-In task https://github.com/coala/coala/wiki/Google-Code-In-Task-Implement-coala .

The mentors are : @jayvdb 
In addition to normal review comments, collaborators are welcome to come and discuss the task at https://gitter.im/coala/coala .

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -0,0 +1,72 @@
+[sql]
+bears = InvalidLinkBear
+files = **.sql
+
+[volt]
+bears = InvalidLinkBear, LineLengthBear, SpaceConsistencyBear
+files = **.volt
+max_line_length = 200
+use_spaces = True
+enforce_newline_at_EOF = True
```
Comment (jayvdb) [inline .coafile]: this, and `use_spaces`, should be in the Default config section, so they do not need to be repeated.

File: .coafile [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+[php-PascalStyleFilenameing]
+bears = FilenameBear
+files = app/*.php, app/controllers/**.php, app/forms/**.php, app/library/**.php, app/models/**.php, app/plugins/**.php
+file_naming_convention = pascal
+
+[php-public]
+bears = InvalidLinkBear, PHPLintBear, LineLengthBear, LineCountBear
+files = public/index.php
+max_line_length = 120
```
Comment (jayvdb) [inline .coafile]: this looks like a good default.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+language: generic
+services: docker
+script:
+  - >
+      docker run --volume=$(pwd):/app --workdir=/app coala/base:pre /bin/bash
+      -c "pip3 install 'git+https://github.com/jayvdb/coala-quickstart@gh_39';
```
Comment (jayvdb) [inline .travis.yml]: `coala-quickstart` not needed.

File: .coafile [hunk 4]
```diff
@@ -0,0 +1,72 @@
+[sql]
```
Comment (jayvdb) [inline .coafile]: Create a `[Default]` section at the top, and put common config there, like:

```
[Default]
use_spaces = True
enforce_newline_at_EOF = True
max_line_length = 120
indent_size = 4
```

**Comment by sergeysviridenko** [comment]
All phalcon projects using standard instruments like travis ci. Sorry but I think this PR can't be implemented.
@sergeyklay I propose close this PR.

**Pull request closed without merging**